src/commentsClass.js

```diff
@@ -0,0 +1,64 @@
+const home = document.querySelector('.home');
+
+export default class TVmazeAPI {
+  constructor() {
+    this.url = 'https://api.tvmaze.com/shows';
+  }
+
+  async displayShow(itemId) {
+    const response = await fetch(this.url);
+    const data = await response.json();
+
+    const item = data.find((item) => item.id === itemId);
+
+    const popupDiv = document.createElement('div');
+
```

Comment:
i suggest use api `https://api.tvmaze.com/shows/:id` instead use the function `data.find `